modules/pubmaticServerBidAdapter.js

```diff
@@ -286,45 +351,19 @@ export const spec = {
   * Register User Sync.
   */
   getUserSyncs: (syncOptions, serverResponses, gdprConsent) => {
-    let serverResponse;
     let urls = [];
-    // Todo: Can fire multiple usersync calls if multiple responses for same adsize found
-    if (serverResponses.length > 0 && serverResponses[0] && serverResponses[0].body) {
-      serverResponse = serverResponses[0].body;
-    }
-    if (serverResponse && serverResponse.ext && serverResponse.ext.bidderstatus && utils.isArray(serverResponse.ext.bidderstatus)) {
-      serverResponse.ext.bidderstatus.forEach(bidder => {
-        if (bidder.usersync && bidder.usersync.url) {
-          // Attaching GDPR Consent Params in UserSync urls
-          if (gdprConsent) {
-            bidder.usersync.url += '&gdpr=' + (gdprConsent.gdprApplies ? 1 : 0);
-            bidder.usersync.url += '&gdpr_consent=' + encodeURIComponent(gdprConsent.consentString || '');
-          }
-
-          if (bidder.usersync.type === IFRAME) {
-            if (syncOptions.iframeEnabled) {
-              urls.push({
-                type: IFRAME,
-                url: bidder.usersync.url
-              });
-            } else {
-              utils.logWarn(bidder.bidder + ': Please enable iframe based user sync.');
-            }
-          } else if (bidder.usersync.type === IMAGE || bidder.usersync.type === REDIRECT) {
-            if (syncOptions.pixelEnabled) {
-              urls.push({
-                type: IMAGE,
-                url: bidder.usersync.url
-              });
-            } else {
-              utils.logWarn(bidder.bidder + ': Please enable pixel based user sync.');
-            }
-          } else {
-            utils.logWarn(bidder.bidder + ': Please provide valid user sync type.');
-          }
-        }
-      });
-    }
+    var bidders = config.getConfig('userSync.enabledBidders');
+    var UUID = utils.getUniqueIdentifierStr();
+    var data = {
+      uuid: UUID,
+      bidders: bidders
+    };
+    ajax.ajax(COOKIE_SYNC, cookieSyncCallBack(gdprConsent), JSON.stringify(data), {
+      withCredentials: true,
+      customHeaders: {
+        'Access-Control-Allow-Origin': '*'
```

Comment:
This is a response header, why are we sending it in a request?
Refer: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin